zuul.d/project.yaml

```diff
@@ -4,57 +4,10 @@
       - ansible-python-jobs
     check:
       jobs:
-        - ansible-network-asa-appliance:
```

Comment:
there is no need to delete these jobs, they will only run is files specific to them change.